Dockerfile

```
@@ -44,6 +44,8 @@ RUN set -eux; \
     chmod -R g+rwX /home/node; \
     chown -R 1000 /home/node
 
+RUN apk add --no-cache libcap && setcap 'cap_net_raw+ep' $(which node)
```

Please adjust the script to make it more robust, secure and easier to debug

```suggestion
RUN set -eux; \
    apk upgrade --no-cache; \
    apk add --no-cache libcap; \
    rm -rf /var/cache/apk/*
    
RUN setcap 'cap_net_raw+ep' $(which node)
```